NashTeamAlpha/BangazonWeb
# Jd single type product list

## Pull Request Description

Include the applicable information from the list below here 

1. Completed ProductsInSubTypeView
1. Added hotfix to Single view to remove extra dollar sign

## Steps to Test

Include the applicable information from the list below here

1. Switch to branch jd-singleTypeProductList
1. Add a new product, and verify that it shows up in the appropriate sub category, or look for an existing product.

## Link to Feature Ticket

Provide the issue number that this pull request fixes
Example: #109
